### ISSUE TITLE
show dot1x all details -> Missing keys: [['interfaces', 'GigabitEthernet1/6', 'clients', '6451.065c.f902', 'eap_method']]

### ISSUE THREAD

**Issue by urskog84** [CiscoTestAutomation/genieparser#64]
I have - > EAP Method                = (13)

```
Dot1x Info for GigabitEthernet1/6
-----------------------------------
PAE                       = AUTHENTICATOR
QuietPeriod               = 3
ServerTimeout             = 0
SuppTimeout               = 15
ReAuthMax                 = 2
MaxReq                    = 1
TxPeriod                  = 1

Dot1x Authenticator Client List
-------------------------------
EAP Method                = (13)
Supplicant                = 6451.065c.f902
Session ID                = 0A90740B0000A7FD44A5F6A8
    Auth SM State         = AUTHENTICATED
    Auth BEND SM State    = IDLE
```

**Comment by thaixuann** [comment]
Hello @urskog84 ,

We fixed the issue and genie package v19.11.1b1 has been released with the fix.
Could you please run the following command and try again to see if it works?

pip install --upgrade --pre genie.libs.parser,

Please try and let's know if there is an issue.

Thanks,
--Thai.

**Comment by urskog84** [comment]
Thanks,
